src/pages/Hub.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from "react";
+import { Container, Row, Col, Button } from "react-bootstrap";
+import { Link } from "react-router-dom";
+
+const Hub = (props) => {
```

Comment:
```suggestion
const Hub = ({ userInSession }) => {
```

We can *extract* specific properties from a props object. This also makes the code more readable, as the the expected properties are document. If you only want to this for specific properties, you would use `{ specificProperty, ...props }`